src/Passes/Source/Passes/QubitAllocationAnalysis/QubitAllocationAnalysis.hpp

```diff
@@ -15,29 +15,13 @@ namespace quantum
     class QubitAllocationAnalysisAnalytics : public llvm::AnalysisInfoMixin<QubitAllocationAnalysisAnalytics>
```

Comment:
A comment here on the class regarding what it is/does would be good to have. 
In general, the policy (enforced upon code review for PRs to master) is that the public API should be fully documented - in the future, I'd expect we will ultimately automatically generate the API docs for C++ code.
The same goes for other files/classes.